experiments/githubcloner.sh

```diff
@@ -0,0 +1,41 @@
+echo 'Enter github repository link'
+read LINK 
```

Comment:
Instead of asking the user to type this information in line by line, I would recommend you to change the script so that you pass all of this information in as arguments when the script is invoked. E.g., bash githubcloner.sh LINK DIR_NAME SHA...
This would allow your script to be usable by other scripts such as one that can clone multiple repositories at once with a predefined list of LINK, DIR_NAME, SHA, ...